lxd-agent/exec.go

```diff
@@ -414,12 +425,25 @@ func (s *execWs) Do(op *operations.Operation) error {
 	cmd.Stdin = stdin
 	cmd.Stdout = stdout
 	cmd.Stderr = stderr
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		Credential: &syscall.Credential{
```

Comment:
This should be taken from `https://godoc.org/golang.org/x/sys/unix` not from `syscall`, no?

Reply:
Ah yes "Deprecated: this package is locked down. Callers should use the corresponding package in the golang.org/x/sys repository instead."

Reply:
`exec.Command` depends on `syscall.SysProcAttr` directly, we don't get to use unix for this one

Reply:
_facepalm_